### PR TITLE
Remove all cumulus/db releated tests/CI/TF

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-2180: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2180)

# Changes

* Removes all RDS DB related CI/TF/code from Core master in favor of the DB branch
* Leaves unpublished cluster creation module, as that shouldn't impact the release. 

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -140,48 +140,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
-    for diagnostic purposes, and retry certain connection timeouts once.
-    This option is disabled by default
+    severless RDS Aurora/ PostgreSQL database cluster to meet the PostgreSQL
+    requirements for future releases
 - **CUMULUS-2156**
   - Support array inputs parameters for `Internal` reconciliation report
-- **CUMULUS-2157**
-  - Added support to `data-migration1` Lambda for migrating providers data from Dynamo to RDS
-    - The migration process for providers will convert any credentials that are stored unencrypted or encrypted with an S3 keypair provider to be encrypted with a KMS key instead
-- **CUMULUS-2161**
```
Comment (markdboyd) [inline CHANGELOG.md:171]: CHANGELOG for 2161 should not have been removed?

**Review by markdboyd — CHANGES_REQUESTED** [review]
One minor requested change

File: CHANGELOG.md (hunk 1 above)
Comment (Jkovarik) [reply CHANGELOG.md:171]: Ugh, yeah, probably a bad merge.  Will sort.

Comment (markdboyd) [reply CHANGELOG.md:171]: Sorry, still looks off? I wouldn't think we should have a diff for this CHANGELOG section

Comment (Jkovarik) [reply CHANGELOG.md:171]: Updated.   The auto-line-length setup made a few changes to the entry, we should probably be doing that anyway.

Comment (Jkovarik) [reply CHANGELOG.md:171]: 9027c61d1 and b82c676d6

**Review by markdboyd — APPROVED** [review]

**Pull request merged**